### PR TITLE
Update the check for MultipleOperationsFound

### PR DESCRIPTION
**Pull request by lirenhe**
Improve the error handling and update the test case

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "oav",
-  "version": "0.20.4",
+  "version": "0.20.5-rc",
```
Comment (raych1) [inline package.json]: Use beta for testing purpose instead of rc.

File: test/liveValidatorTests.ts [hunk 2]
```diff
@@ -673,6 +673,29 @@ describe("Live validator snapshot validation", () => {
...
+      }
+    }
+    const liveValidator = new LiveValidator(options)
+    await liveValidator.initialize()
+
+    const payload = require(`${__dirname}/liveValidation/payloads/multiplePperationFound_input`)
+    const result = liveValidator.validateLiveRequestResponse(payload)
+    expect(
+      result.responseValidationResult.runtimeException &&
+        result.responseValidationResult.runtimeException.code === "MULTIPLE_OPERATIONS_FOUND"
```
Comment (raych1) [inline test/liveValidatorTests.ts]: Can we verify the error message contain the specPath?

File: test/liveValidation/swaggers/specification/mediaservices/resource-manager/Microsoft.Media/2019-05-01-preview/Common.json [hunk 3]
```diff
@@ -0,0 +1,80 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Media Services",
+    "version": "2018-07-01"
```
Comment (raych1) [inline test/liveValidation/swaggers/specification/mediaservices/resource-manager/Microsoft.Media/2019-05-01-preview/Common.json]: Should this version be same as folder name?

Comment (lirenhe) [reply test/liveValidation/swaggers/specification/mediaservices/resource-manager/Microsoft.Media/2019-05-01-preview/Common.json]: No, it is to create an issue of multiple operations in the same API version and this is actually a real case.

**Pull request merged**